critiquebrainz/frontend/views/review.py

```diff
@@ -94,17 +97,27 @@ def entity(id, rev=None):
             vote = None
     else:  # otherwise set vote to None, its value will not be used
         vote = None
-    review["text_html"] = markdown(revision['text'], safe_mode="escape")
+    if revision["text"] is None:
+        review["text_html"] = None
+    else:
+        review["text_html"] = markdown(revision['text'], safe_mode="escape")
 
     user_all_reviews, review_count = db_review.list_reviews(  # pylint: disable=unused-variable
         user_id=review["user_id"],
         sort="random",
         exclude=[review["id"]],
     )
     other_reviews = user_all_reviews[:3]
+    if review["rating"] is not None:
+        review["rating"] //= 20
```

Comment:
I'm still interested in a fix for this. Conversion from 5-star to 100 should be done within the `db` package. Modules that use it don't need to know about how ratings are stored. It's about making the interface less complex and easier to use.